ios/brave-ios/Sources/Favicon/FaviconFetcher.swift

```diff
@@ -161,6 +161,13 @@ public class FaviconFetcher {
     await storeInCache(favicon, for: url, persistent: persistent)
   }
 
+  /// Delete the favicon from disk and memory cache for the given URL
+  public static func deleteCache(for url: URL) async {
+    let cachedURL = cacheURL(for: url)
+    SDImageCache.shared.memoryCache.removeObject(forKey: cachedURL.absoluteString)
+    SDImageCache.shared.diskCache.removeData(forKey: cachedURL.absoluteString)
```

Comment:
Do we have to wrap these in a Task to enforce background thread usage?

Reply:
No its `nonisolated` by default because `FaviconFetcher` is a class that doesn't have any MainActor isolation and it has `async`